spec/use_cases/schema2.0.0/L000_Prelim_Analysis.sch

```diff
@@ -1,103 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we still want to keep our previous implementations for schematron validation?

Reply:
Unless you are planning on regenerating this using all of the new stuff

Reply:
I think we'll just use the newer version at v2.1.0/L000_Prelim_Analysis.sch, though I might be misunderstanding your question

Reply:
The notion here was to keep all current v2.0.0 schematron implementations.  This has been added in `TestSuite/schematron/v2.0.0`.